absearch/settings.py

```diff
@@ -144,8 +146,9 @@ def get(self, prod, ver, channel, locale, territory, dist, distver,
         locale = _lower(locale)
         territory = _lower(territory)
         prod = _lower(prod)
+        # verify version at least contains a number
         try:
-            ver = int(ver.split('.')[0])
+            int(ver)
```

Comment:
I don't understand this change, which seems a bit counter-intuitive as per the PR title :|

Reply:
The reason for that change is because LooseVersion allows .* as a version, so the test fails. This is effectively a sanity check to make sure the version at least contains a number.

I looked into using StrictVersion, but it doesn't allow versions without a "." so that breaks the existing code as well.